src/pages/data-plane-nodes/List.vue

```diff
@@ -0,0 +1,399 @@
+const nodeLogLevel = ref<Record<string, LogLevelResponse>>({})
+const hasLogLevelChanged = ref<boolean>(false) // if some log level has been changed
+
+const nodeCurrentLogLevel = computed<Map<string, LogLevel>>(() => {
+  return Object.keys(nodeLogLevel.value).reduce((acc, id) => {
+    acc.set(id, nodeLogLevel.value[id].current_level)
+
+    return acc
+  }, new Map())
+})
+const selectedNodeCount = computed<number>(() => {
+  return visibleNodes.value.filter(node => node.selected).length
+})
+const isAllVisibleNodesSelected = computed<boolean>(() => {
+  return selectedNodeCount.value === visibleNodes.value.length
+})
+const isSomeVisibleNodesSelected = computed<boolean>(() => {
+  return selectedNodeCount.value > 0 && selectedNodeCount.value < visibleNodes.value.length
+})
+
+const fetcher = async (props: FetcherParams) => {
+  errorMessage.value = ''
+  requestScheduler.cancelAll()
+
+  const {
+    page,
+    pageSize,
+    offset,
+  } = props
+
+  try {
+    const { data } = await apiService.findAll<{ data: DataPlaneNodeResponse[]; offset?: number }>(
+      'clustering/data-planes',
+      {
+        size: pageSize,
+        offset: page === 1 ? undefined : offset,
+      },
+    )
+
+    visibleNodes.value = data.data.map(node => ({
+      ...node,
+      hasDLLCapability: node.rpc_capabilities?.includes?.('kong.debug.log_level.v1') ?? false,
+      selected: false,
+    }))
+    scheduleLogLevelRequests()
+
+    return {
+      data: visibleNodes.value,
+      ...(data.offset
+        ? {
+            pagination: {
+              offset: data.offset,
+            },
+          }
+        : null),
+    }
+  } catch (err) {
+    const { response } = err as AxiosError<{ message: string }>
+
+    errorMessage.value = response?.data?.message || t('entities.dp-nodes.error.fetch.fail')
+  }
+}
+
+const scheduleLogLevelRequests = () => {
+  visibleNodes.value.forEach(async node => {
+    if (!node.hasDLLCapability) {
+      return
+    }
+
+    try {
+      await requestScheduler.schedule(() => {
+        return getNodeLogLevel(node.id)
+      })
+    } catch (err) {
+      if (err instanceof composables.AsyncAbortException) {
+        return
+      }
+
+      console.error(err)
+    }
+  })
+}
+
+const onGlobalSelectionChange = () => {
+  const val = !isAllVisibleNodesSelected.value
+
+  visibleNodes.value.forEach(node => {
+    node.selected = val
+  })
+}
+
+const onRowSelectionChange = (id: string) => {
+  const target = visibleNodes.value.find(node => node.id === id)
+  if (target) {
+    target.selected = !target.selected
+  }
+}
+
+const formatDateFromTimeout = (timeout: number) => {
+  return formatDate(Date.now() / 1000 + timeout, 'MMM DD, YYYY, h:mm:ss A')
+}
+
+const openLogLevelEditModal = () => {
+  modalVisible.value = true
+}
+
+const getNodeLogLevel = async (id: string) => {
+  try {
+    const response: AxiosResponse<LogLevelResponse> = await apiService.get(`clustering/data-planes/${id}/log-level`)
```

Comment:
very-tiny-nit: Would we use `await apiService.get<LogLevelResponse>` here?

Reply:
Updated.